integrationtests/persistence/test_persistence.py

```diff
@@ -17,11 +17,11 @@ def __init__(self, firstname, lastname):
 class Foo:
```

Comment:
TBH these integration tests could probably just be deleted.  I don't believe they ever get run/can be run

Reply:
There might be values in converting them to test cases so that they can be discovered and run? In any case, that would be a separate issue. Agreed that tests don't get automatically run typically get forgotten and end up bring broken long before anyone realizes.

Reply:
Yeah that makes sense.  I will leave that for a separate issue/PR